documentation/docs/01-routing.md

```diff
@@ -269,6 +271,13 @@ Most commonly, endpoints exist to provide data to the page with which they're pa
 
 > Support for streaming request and response bodies is [coming soon](https://github.com/sveltejs/kit/issues/3419).
 
+Standalone endpoints can be given a file extension if desired, or accessed directly if not:
+
+| Filename | Endpoint |
+| - | - |
```

Comment:
From https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables:
>There must be at least three hyphens in each column of the header row.

I'm not sure if our flavor of markdown enforces the same rules, but it might be safer to add more here:

```suggestion
| -------- | -------- |
```

Reply:
Prettier takes care of this automatically